### DISCUSSION TITLE
please explain "⚠️ Mempool Limited Yes"

### DISCUSSION THREAD

**Discussion by photis** [Mirobit/bitcoin-node-manager#82] [Q&A] [answered]
I noticed that my node always displays "⚠️ Mempool Limited Yes", regardless of the "maxmempool" value (which I set to 3 GB). Is there something I'm misunderstanding about the mempool here?

**Comment by photis** [comment]
I'm gonna close this discussion as -- wouldn't you know it -- 3 hours after I posted it, the message has disappeared and is now a normal "Mempool Limited No". Sorry for littering.

**Comment by Mirobit** [comment, marked as answer]
@photis If "Mempool limited" shows yes, you can hover over the yes and it should show you how much space is used. It should only display yes if your maxmempool limit is reached.

**Reply by photis** [reply]
Thank you, I *thought* it would work like that, but I was surprised that it didn't react to me increasing the maxmempool size. Only after about 3 hours did it react to that (as I pointed out above). Thanks for taking the trouble to get back to me. And thanks of course for BNM in the first place!
